OpenAI-DotNet/Models/Model.cs

```diff
@@ -28,7 +34,7 @@ public Model(string id)
         /// <summary>
         /// Allows a string to be implicitly cast as a <see cref="Model"/>
         /// </summary>
-        public static implicit operator Model(string name) => new Model(name);
+        public static implicit operator Model(string name) => new(name);
```

Comment:
ugly, but I'll allow it since it is explicit in the first half of the statement lol.